bundles/core-classes/classes/warrior.js

```diff
@@ -10,7 +10,7 @@
 module.exports = srcPath => {
   return {
     name: 'Warrior',
-    description: 'Warriors relish being face-to-face with their enemy. Whether it be wielding axes, maces, swords, or a nearby log, Warriors focus on dealing strong physical damage to their opponent. What they make up for in the raw damage of a mage, or the healing prowess of a Cleric, Warriors make up for in their tenacity. Those choosing the more defensive path of the shield can outlast otherwise deadly attacks.',
+    description: 'Warriors relish being face-to-face with their enemy. Whether it be wielding axes, maces, swords, or a nearby log, Warriors focus on dealing strong physical damage to their opponent. What they make up for in the raw damage of a Mage, or the healing prowess of a Cleric, Warriors make up for in their tenacity. Those choosing the more defensive path of the shield can outlast otherwise deadly attacks.',
```

Comment:
I wrote this and the language seems weird in hindsight. I think I meant to write "What they _lack_ in the raw damage of a Mage..."